pkg/controller/instanceset/instance_util.go

```diff
@@ -651,6 +664,105 @@ func mergeCPUNMemory(s, d *corev1.ResourceList) {
 	}
 }
 
+func mergeAffinity(affinity1Ptr, affinity2Ptr **corev1.Affinity) {
```

Comment:
Seems we have two mergeAffinity now.

Should be addressed in the future pr.

Reply:
@weicao 👍 I didn't find a similar method before, I'll see if I can reuse it directly.

Reply:
> @weicao 👍 I didn't find a similar method before, I'll see if I can reuse it directly.

in pkg/controller/scheduling/scheduling_utils.go

Reply:
The two function implementations are different. The previous implementation is a direct accumulation, and my implementation is merged if the key is the same.

Can a TODO action be added first, and then create a separate PR to merge these two methods? @free6om

Reply:
Sure